mautic/mautic
# [UX] Reorder administration panel

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Reordering administration panel based on discussion: https://mautic.slack.com/archives/C024XBCQS3Z/p1711566166387599



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the right sidebar
3. Look, click on these things to make sure it's working

---

| Before                                      | After
| -------------------------------------- | ---
![image](https://github.com/mautic/mautic/assets/116097999/8858e97e-b3b9-4ba5-9db1-b1515deea5d9) | ![image](https://github.com/mautic/mautic/assets/116097999/b002e981-eb58-4ab7-80ca-d9b003db9cfa)
